src/TestInfrastructure/VsObjectModelMocks/SolutionMock.cs

```diff
@@ -497,7 +497,7 @@ int IEnumHierarchies.Skip(uint celt)
 
         #region Test helpers
 
-        public bool IsFullyLoaded
+        public object IsFullyLoaded
```

Comment:
Why object?

Reply:
The VS API returns an _object_; returning _object_  here means we can test that product code handles conversion failures gracefully.

Reply:
I would add this comment on top of the property (for later).